gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -73,20 +81,38 @@ func NewVMKeeper(
 	return vmk
 }
 
-func (vm *VMKeeper) Initialize(ms store.MultiStore) {
+func (vm *VMKeeper) Initialize(
+	logger *slog.Logger,
+	ms store.MultiStore,
+	cacheStdlibLoad bool,
```

Comment:
If I’m not wrong, it’s not only about caching stdlib, but about enforcing that the whole gnoStore is in memory.
The flag name should eventually be renamed to emphasize that the impact isn’t only on stdlibs.